stable_baselines3/common/buffers.py

```diff
@@ -381,3 +381,118 @@ def _get_samples(self, batch_inds: np.ndarray, env: Optional[VecNormalize] = Non
             self.returns[batch_inds].flatten(),
         )
         return RolloutBufferSamples(*tuple(map(self.to_torch, data)))
+
+
+class NstepReplayBuffer(ReplayBuffer):
```

Comment:
Minor: why not `NStepReplayBuffer`?

Reply:
I used it as a single word. I will update it.